modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -306,10 +319,17 @@ private void calculateLatency(ComplexEventChunk<StreamEvent> streamEventChunk) {
                 StreamEvent streamEvent = streamEventChunk.next();
                 try {
                     long currentTime = System.currentTimeMillis();
-                    long iijTimestamp = (Long) (attributeExpressionExecutors[0].execute(streamEvent));
+
                     timeSpent += (currentTime - iijTimestamp);
-                    eventCount++;
-                    eventCountTotal++;
+
+                    long iijTimestamp = (Long) (attributeExpressionExecutors[0].execute(streamEvent));
+
+
```

Comment:
Remove unnecessary new lines. Check in all places

Reply:
Acknowledged